projects/npm-tools/packages/npm-scripts/src/utils/writeWebpackFederationEntryPoint.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const {transformJsSource} = require('@liferay/js-toolkit-core');
```

Comment:
I think we need to include this package in npm-scripts package.json

Reply:
Yes, I saw it was missing yesterday, but forgot to send the PR. However it always works because in this repo, the package is linked inside the root `node_modules` and, in portal, because we use the bundler, it also appears :sweat_smile: .

But I'll send a PR to fix it :+1: .